packages/sanity/src/core/releases/components/Table/Table.tsx

```diff
@@ -0,0 +1,182 @@
+import {Box, Card, Flex, Stack, Text} from '@sanity/ui'
+import {get} from 'lodash'
+import {Fragment, useMemo} from 'react'
+import {useTableContext} from 'sanity/_singletons'
+import {styled} from 'styled-components'
+
+import {LoadingBlock} from '../../../components'
+import {TableHeader} from './TableHeader'
+import {TableProvider} from './TableProvider'
+import {type Column} from './types'
+
+export interface TableProps<D, AdditionalRowD> {
```

Comment:
This is somewhat stylistic, but what do you think about replacing the abbreviation `D` with a more descriptive name?

Reply:
Yeh that's totally fair. One minute I'm explicit with `datum`, the next just `D` haha!

- [ ] update generic param names to be more descriptive